docs/Encyclopedia/pages/TCGA_VCF_1.1v2.md

```diff
@@ -1297,12 +1286,11 @@ UUID-compliant files should satisfy the following criteria:
 2. If ##INDIVIDUAL is declared in the header, then values assigned to
 SampleUUID in all ##SAMPLE declarations should correspond to the same
 participant ID, and the TCGA barcode for the participant should be assigned to
-##INDIVIDUAL.
 
+## INDIVIDUAL.
 
+### Handling failed checks

```

Comment:
I _think_ this and "Test files" below could be ##, but I don't feel too strongly about it. Check with Bill.

Reply:
Let's leave it as is and change if it deploys weird.